CONTRIBUTING.md

```diff
@@ -9,14 +9,28 @@ document explains how to contribute successfully.
 
 #### Development Branch
 
-Most work should start with the latest instance of the `main` branch and be performed in a development branch:
+Implementation of new features will be tracked in the `main` branch while bug
+fixes and doc changes for the latest release will be tracked in the maintenance
+branch, called `maint-RELEASE_VERSION`. All work will be performed in a
+new development branch starting with the appropriate base branch and merged
+back into that branch.
+
+For new features:
 
 ```console
 git checkout main
 git pull
 git checkout -b new-branch-name
 ```
 
+For bug fixes and doc changes to release version 2.0.1:
```

Comment:
```suggestion
For example, for bug fixes and doc changes to release version 2.0.1:
```

Reply:
I'd rather say we say "for example" here, unless we want to change this contributing doc every time we have a new release.

Reply:
LOL no we do not. thank you.